keyboards/mss_studio/m64_rgb/keymaps/default/keymap.c

```diff
@@ -0,0 +1,188 @@
+/* Copyright 2021 Mss Studio
+ * Copyright 2022 HorrorTroll <https://github.com/HorrorTroll>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <string.h>
+#include <math.h>
+
+#include <lib/lib8tion/lib8tion.h>
+
+#include QMK_KEYBOARD_H
```

Comment:
```suggestion
#include QMK_KEYBOARD_H
#include <lib/lib8tion/lib8tion.h>
#include <math.h>
```

lib8tion.h includes string.h, but only if it's not arm. 
Also, it includes math8.h from it's own code, which may make math.h unnecessary, as well.